.github/eks-workflow-files/jio-eks/jio-template.yaml

```diff
@@ -0,0 +1,549 @@
+                RAY_PID=$!
+
+                sleep 10  # Give time to Ray to settle
+
+                # Start vLLM Rollout Controller
+                echo "Starting vLLM Rollout..."
+                cd /opt/jtbx/jax-inference-offloading/examples
+                export RAY_ADDRESS="${HOST_IP}:${RAY_PORT}"
+                CUDA_VISIBLE_DEVICES="" ray status
+                python -u rollout.py 2>&1 | tee rollout.log &
+                ROLLOUT_PID=$!
+
+                wait ${ROLLOUT_PID}
+                ROLLOUT_EXIT=$?
+                echo "Rollout.py finished with code ${ROLLOUT_EXIT}"
+                ray stop --force || truekill ${RAY_PID}
+                kill ${RAY_PID} 2>/dev/null || true
+                echo "VLLM completed"
+                exit ${ROLLOUT_EXIT}
+              env:
+              - name: MY_HOST_IP
+                valueFrom:
+                  fieldRef:
+                    fieldPath: status.hostIP
+              - name: JOB_COMPLETION_INDEX
+                valueFrom:
+                  fieldRef:
+                    fieldPath: metadata.annotations['batch.kubernetes.io/job-completion-index']
+              - name: JOBSET_NAME
+                valueFrom:
+                  fieldRef:
+                    fieldPath: metadata.annotations['jobset.sigs.k8s.io/jobset-name']
+
+              # General config
+              - name: HF_TOKEN
+                valueFrom:
+                  secretKeyRef:
+                    name: huggingface-token
+                    key: token
+              - name: MODEL_NAME
+                value: PLACEHOLDER # here we need MODEL
+              - name: MODEL_PATH
+                value: ""
+              # EFA
+              - name: FI_PROVIDER
+                value: "efa"
+              # General VLLM
+              - name: TORCH_CUDA_ARCH_LIST
+                value: "9.0"
+              # Gateway
+              - name: GATEWAY_PORT
+                value: "50051"
+              - name: GATEWAY_URL
+                value: "jax-vllm-multinode-gateway-0-0.jax-vllm-multinode:50051"
+              # Ray settings
+              - name: RAY_PORT
+                value: "20527"
+              - name: RAY_CLIENT_SERVER_PORT
+                value: "24430"
+              # CUDA settings
+              - name: CUDA_DEVICE_ORDER
+                value: "PCI_BUS_ID"
+              - name: CUDA_DEVICE_MAX_CONNECTIONS
+                value: "16"
+              - name: CUDA_VISIBLE_DEVICES
+                value: "0,1,2,3,4,5,6,7"
+              # vLLM settings
+              - name: VLLM_ENFORCE_EAGER
+                value: "1"
+              - name: VLLM_LOAD_FORMAT
+                value: "dummy"
+              - name: VLLM_GPU_MEMORY_UTILIZATION
+                value: "0.4"
+              - name: VLLM_TENSOR_PARALLEL_SIZE
+                value: "8"
+              - name: VLLM_DISTRIBUTED_BACKEND
+                value: "ray"
+              - name: NCCL_CUMEM_ENABLE
```

Comment:
Let's remove this one. It is fine for Hopper but hurts perf on Blackwell. It is [no longer needed after vLLM  0.10.1](https://docs.vllm.ai/en/v0.11.2/usage/troubleshooting/#known-issues).